openforcefield/topology/topology.py

```diff
@@ -1383,6 +1382,9 @@ def from_openmm(cls, openmm_topology, unique_molecules=None):
             OpenMM ``Topology``, these will be used in matching as well.
             If all bonds have bond orders assigned in ``mdtraj_topology``, these bond orders will be used to attempt to construct
             the list of unique Molecules if the ``unique_molecules`` argument is omitted.
+        load_box_vectors : bool. Default=False
```

Comment:
Why would this be exposed as an optional argument? The OpenMM `Topology` object only contains box vectors if the system is meant to be simulated in the condensed phase, so we should presumably pull these automatically if they are present. So I think this behavior should be automatic, and not something we need to expose to the users.

That is, the behavior should be 
* if OpenMM `Topology` has no box vectors, assume system is vacuum
* if OpenMM `Topology` has box vectors, assume system is condensed phase and copy box vectors

We can have API methods for our OFF `Topology` that allow us to modify the system to be condensed phase or vacuum, or alter the box vectors, later.

Reply:
> The OpenMM Topology object only contains box vectors if the system is meant to be simulated in the condensed phase, so we should presumably pull these automatically if they are present.

I'm flexible on this. My experience as a user in setting up tests was that many of our PDB files had some arbitrary values at the top, under the `CRYST1` field. These were, to my surprise, being interpreted by OpenMM as box vectors. 

Thinking more, _we're_ not responsible for the behavior of OpenMM's PDB reader. I think you're right that we shouldn't expose this, and should by default load the box vectors if they're present.

Reply:
Changed to your suggestion in fc031bb